prereqs.adoc

```diff
@@ -22,26 +22,36 @@ on your machine.
 
 == AWS IAM Permissions
 
-If you already have an AWS Account, you need to create an IAM user and attach these http://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies.html[IAM policies].
+If you already have an AWS Account, you need to create an IAM user here: https://console.aws.amazon.com/iam/home?#/users$new
+Just give it a useful name (e.g. `workshop`) and make sure you check the `Programmatic access` box:
 
-    AmazonEC2FullAccess
-    AmazonRoute53FullAccess
-    AmazonS3FullAccess
-    IAMFullAccess
-    AmazonVPCFullAccess
+image::images/add-iam-user.png[]
+
+In the next screen select `Attach existing policies directly` and search for `AmazonEC2FullAccess AmazonRoute53FullAccess AmazonS3FullAccess IAMFullAccess AmazonVPCFullAccess` - it will now show the roles needed: http://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies.html[IAM policies]
+
+image::images/add-policies-to-iam-user.png[]
+
+Check all five and hit next.
 
 Please review this link:https://github.com/kubernetes/kops/blob/master/docs/aws.md#setup-iam-user[link]
 for additional info on IAM permissions
 
-== AWS Region and Availability Zones
+== Configure the AWS CLI
```

Comment:
there are possibly some broken links because of changed header. Can you make sure it all looks good?

Reply:
I have searched for the old header but couldn´t find any results